DiscordChatExporter.Core.Rendering/Resources/HtmlShared.css

```diff
@@ -1,5 +1,35 @@
 /* === GENERAL === */
 
+@font-face {
+    font-family: Whitney;
+    src: url(https://discordapp.com/assets/6c6374bad0b0b6d204d8d6dc4a18d820.woff);
```

Comment:
Are you sure these URLs are static? Is there a way to embed them or would that make the report too big?

Reply:
I'm unfortunately unfamiliar with `@font-face`, what would happen if the user doesn't have internet connection? Will it just fall back to the available fonts for this font family?

Reply:
Good question. I'm 100% sure that they're static, but I can't find a more reliable looking url to use. We could potentially host them on the repository's pages website thing? Idk. 

They're ~80KB each, and since a simple export for a channel is like 40KB, it's probably not worth it to embed these in the report. (80 * 5) = 400KB of extra data.

Good news is that yes `font-family` will just fall back to the next available font. In this case if it fails to get the Whitney, it will fall back to Arial like before. (I tested this to confirm by just typing stuff in the urls to make then invalid.)

Reply:
Awesome, I think this is good enough then